### PR TITLE
Update build info and qemu config

### PR DESCRIPTION
**Pull request by crawfxrd**
- `models/qemu/edk2.config`: Ensure QEMU builds with correct edk2 configs
- `scripts/qemu.sh`: Change `kvm` to `qemu-system-x86_64`
- `scripts/deps.sh`: Update `rustup` or inform users to source new env vars
- `readme.md`: Update info for building

### CODE REVIEW

**Comment by 13r0ck** [comment]
I want to wait on approval hear until I get your input on [this](https://github.com/system76/ec/pull/267#issuecomment-1013395055)

**Comment by 13r0ck** [comment]
I get the error `./scripts/qemu.sh: line 20: qemu-system-x86_64: command not found`. I am running the script inside of a `gnome-boxes` virtual machine, am I doing something wrong?

Everything else is fine though

**Comment by crawfxrd** [comment]
Is qemu installed? It's not installed as part of the deps, so you have to install it separately.

**Comment by 13r0ck** [comment]
`apt policy qemu` -> `Installed: 1:6.0+dfsg-2expuntu1.1` yep it's installed. Same issue

Should the README mention that qemu might need to be installed?

**Comment by crawfxrd** [comment]
Okay, apparently the `qemu` package doesn't do shit on Ubuntu. Is `qemu-system-x86` installed?

> Should the README mention that qemu might need to be installed?

I started a doc for using QEMU for dev/debug, but haven't pushed it yet. I will add the package info in that and link it in the README where we use it as an example.

**Review by jackpot51 — DISMISSED** [review]

**Review by jackpot51 — APPROVED** [review]

File: scripts/deps.sh
Comment (crawfxrd) [inline scripts/deps.sh:135]: Maybe I should colorize this output as well. People will probably *still* ignore it because it's white text that isn't the last line of output. Yellow should be good?

**Review by jackpot51 — DISMISSED** [review]

File: README.md
Comment (jacobgkau) [inline README.md:71]: Can we link `QEMU model` on this line to `debugging.md#using-qemu`? (Or link it somewhere else/include the qemu package somewhere else.)

Comment (crawfxrd) [reply README.md]: Done.

**Review by jacobgkau — APPROVED** [review]
Instructions work on a fresh machine.

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**
